src/js/components/RangeSelector/doc.js

```diff
@@ -72,7 +72,27 @@ export const doc = RangeSelector => {
 };
 
 export const themeDoc = {
-  'global.colors.controls': {
+  'global.colors.focus': {
+    description: 'TThe color when there is a focus',
+    type: 'string',
+    defaultValue: 'accent-1',
+  },
+  'global.edgeSize': {
+    description: 'The possible sizes for the margin, padding and gap',
+    type: 'object',
+    defaultValue: 'small',
```

Comment:
`6px` I believe, but please double check. you had a similar prop on your Tabs PR, and you presented it the right way over there.